docs/rules/max-classes-per-file/index.html

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Sorry about that, I'll remove it.  last time I made a PR to this repo these weren't a thing, so it must be new (but I like it!)